index.test.js

```diff
@@ -42,6 +42,14 @@ it('works with postcss-each', () => {
   )
 })
 
+it('works with postcss-hexrgba', () => {
+  runWithPlugins(
+    [plugin(), require('postcss-hexrgba')],
+    '$color-red: #FF0000; .a { color: rgba($color-red,0.10); }',
+    '.a { color: rgba(255,0,0,0.10); }'
```

Comment:
The second line is the expected output, but on 6.0.3 is outputs `.a { color: rgba(#FF0000,0.10); }`